### PR TITLE
Update docs to address the CEIP prompt in automation

### LINKED ISSUE #210: update the docs to provide deatils of CEIP prompt on first usage and how to set it without being prompted
**Describe the feature request**
The CEIP participation prompt was introduced in the latest release  0.90.0-alpha.0. This CEIP particiapation prompt would be shown to users on the first tanzu command usage except `tanzu version`, `tanzu ceip-participation set` and `tanzu config set` commands. 

For user installing the CLI through CI or by any automation means, user should be able to skip the prompt by calling `tanzu ceip-participation set <true/false>` as their first command to set the CEIP participation status and there after wouldn't get prompted for the same.

The above behavior should be mentioned in the docs.

**Describe alternative(s) you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by prkalle**
### What this PR does / why we need it
This PR updates docs to provide information on how to address the CEIP prompt in automation (ecx: CI jobs)
### Which issue(s) this PR fixes

Fixes #210 

### Describe testing done for PR

`make lint` passed

### Release note

```release-note
N/A
```

### CODE REVIEW

File: docs/quickstart/install.md [hunk 1]
```diff
@@ -83,3 +83,11 @@ EOF
 
 sudo yum install tanzu-cli # dnf install can also be used
 ```
+
+## Continuous Integration
+
+****Customer Experience Improvement Program (CEIP) Prompt**** :
+CLI would prompt users for CEIP participation on their first tanzu command usage except `tanzu version`
```
Comment (marckhouzam) [inline docs/quickstart/install.md:90]:
```suggestion
The Tanzu CLI prompts the user for CEIP participation on their first use of any `tanzu` command
```

File: docs/quickstart/install.md [hunk 2]
```diff
@@ -83,3 +83,11 @@ EOF
...
+, `tanzu ceip-participation set` and `tanzu config set` commands.
```
Comment (marckhouzam) [inline docs/quickstart/install.md:91]:
```suggestion
except for `tanzu version`, `tanzu ceip-participation set` and `tanzu config set`.
```

File: docs/quickstart/install.md [hunk 3]
```diff
@@ -83,3 +83,11 @@ EOF
...
+Users installing/using the CLI through CI or by any automation means, should be able to skip the prompt by
```
Comment (marckhouzam) [inline docs/quickstart/install.md:92]:
```suggestion
Users installing/using the CLI through CI or by any automation means should be able to skip the prompt by
```

**Review by marckhouzam — APPROVED** [review]
This looks good @prkalle.
I just added some grammatical suggestions.

File: docs/quickstart/install.md (hunk 1 above)
Comment (prkalle) [reply docs/quickstart/install.md]: Updated. Thanks @marckhouzam .

File: docs/quickstart/install.md (hunk 2 above)
Comment (prkalle) [reply docs/quickstart/install.md]: updated.

File: docs/quickstart/install.md (hunk 3 above)
Comment (prkalle) [reply docs/quickstart/install.md]: Updated

**Review by marckhouzam — APPROVED** [review]
LGTM
Thanks @prkalle

**Pull request merged**